wstakemgr/db.go

```diff
@@ -1153,6 +1153,48 @@ func duplicateExistsInInvalTickets(record *chainhash.Hash,
 	return false
 }
 
+// removeStakePoolInvalUserTickets removes the ticket hash from the inval
+// ticket bucket
+func removeStakePoolInvalUserTickets(ns walletdb.ReadWriteBucket, scriptHash [20]byte,
```

Comment:
They are being removed from the invalid since they are no longer considered invalid due to intervention by the stakepool operator